### ISSUE TITLE
Test for create and execute requests to send to a node

### ISSUE THREAD

**Issue by deyanzz** [hashgraph/hedera-sdk-cpp#164]
Invoking of method `execute()` should be tested using a [Hedera Local Node](https://github.com/hashgraph/hedera-local-node) for the following classes:

- `AccountBalanceQueryIntegrationTest`
- `AccountCreateTransactionIntegrationTest`
- `TransactionReceiptQueryIntegrationTest`
- `TransactionRecordQueryIntegrationTest`
- `TransferTransactionIntegrationTest`

See the [README.md](https://github.com/hashgraph/hedera-local-node/blob/main/README.md) for the requirement how to install the local node.

**Comment by deyanzz** [comment]
Current implementation is not working well. Will be back to finish it once a new PR is opened for [Issue 163](https://github.com/hashgraph/hedera-sdk-cpp/issues/163).

**Comment by deyanzz** [comment]
To be able to make a working test for `AccountCreateTransaction`, first we need to fix [Issue #192](https://github.com/hashgraph/hedera-sdk-cpp/issues/192).

**Comment by deyanzz** [comment]
This issue is currently blocked by missing capabilities in `Client` - see [Issue #201](https://github.com/hashgraph/hedera-sdk-cpp/issues/201).
Once [Issue #201](https://github.com/hashgraph/hedera-sdk-cpp/issues/201) is merged into main branch the integration tests could be finished.

**Comment by deyanzz** [comment]
This issue is blocked by [Issue #257](https://github.com/hashgraph/hedera-sdk-cpp/issues/257).

**Comment by deyanzz** [comment]
Continue working on this issue as [issue #257](https://github.com/hashgraph/hedera-sdk-cpp/issues/257) and [issue #201](https://github.com/hashgraph/hedera-sdk-cpp/issues/201) were merged into `main` branch.
